### PR TITLE
Add OSMS data into makers dataset.

### PR DESCRIPTION
**Pull request by awong-dev**
Currently it's a csv so just parsing locally, doing the geocode
and then appending the dataset. This also adds a proper rate limit
to the geocoding API from google maps so we don't trigger the abuse
limit.

### CODE REVIEW

File: functions/airtable-connector.js [hunk 1]
```diff
@@ -70,7 +73,7 @@ function toDataJson(entries) {
   };
 }
 
-function loadMakerData(admin, req, res) {
+function loadFromAirtable(static_data, admin, req, res) {
   const base = new Airtable({apiKey: 'key7sP7fMv2drjUnJ'}).base('appmMuprpuCFw4wCd');
```
Comment (nburt) [inline functions/airtable-connector.js:77]: config vars -> google cloud env? Should we also create a new key?

File: functions/airtable-connector.js [hunk 2]
```diff
@@ -1,6 +1,9 @@
 const Airtable = require('airtable');
 const functions = require('firebase-functions');
+const csv = require('csv-parse');
+const fs = require('fs');
 const zipcodes = require('zipcodes');
+const { geocodeAddress } = require('./geocode.js');
```
Comment (nburt) [inline functions/airtable-connector.js:6]: is this file checked in?

Comment (awong-dev) [reply functions/airtable-connector.js:6]: oops. checked in.

File: functions/airtable-connector.js (hunk 1 above)
Comment (awong-dev) [reply functions/airtable-connector.js]: moved. Will generate new key later.

**Pull request merged**